### PR TITLE
Expires header

### PR DESCRIPTION
**Pull request by GoldenGnu**
Fix for issue  #38

### CODE REVIEW

File: src/main/java/net/troja/eve/esi/HeaderUtil.java [hunk 1]
```diff
@@ -43,21 +45,20 @@ public static Integer getXPages(Map<String, List<String>> responseHeaders) {
      *            ApiClient.getResponseHeaders()
      * @return
      */
-    public static Date getExpires(Map<String, List<String>> responseHeaders) {
+    public static OffsetDateTime getExpires(Map<String, List<String>> responseHeaders) {
         // Get header
         String header = getHeader(responseHeaders, EXPIRES);
         if (header == null) {
+            System.out.println("Header Null");
```
Comment (burberius) [inline src/main/java/net/troja/eve/esi/HeaderUtil.java]: Remove this

File: src/test/java/net/troja/eve/esi/HeaderUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    public void testNoDayNoSec() {
+        test("13 Mar 2010 11:29 -0800");
+    }
+
+    private void test(String testValue) {
+        Map<String, List<String>> responseHeaders = Collections.singletonMap("Expires", Collections.singletonList(testValue));
+        OffsetDateTime expires = HeaderUtil.getExpires(responseHeaders);
+        assertThat(expires, notNullValue());
+    }
+
```
Comment (burberius) [inline src/test/java/net/troja/eve/esi/HeaderUtilTest.java]: Add also a test that fails and returns null
